apache/trafficcontrol
# Missing user in logs bug

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

To test this change, you will need to test an API call to TO using 3 different authentication methods(mojolicious, access token, and bearer token).

- Login to TO and obtain mojolicious cookie and access token.
- Observe access.logs while making next requests
- Create request with mojolicious cookie.  
    - Ex: curl -k --cookie "mojolicious=`mojolicious cookie`" -H "Content-Type: application/json" "`your TO endpoint`"
- Create request with access token.  
    - Ex: curl -k --cookie "access_token=`access token`" -H "Content-Type: application/json" "`your TO endpoint`"
- Create request with access token.  
    - Ex: curl -k --cookie "Authorization: Bearer `access token`" -H "Content-Type: application/json" "`your TO endpoint`"
- Ensure with each request that in access.logs logs the correct user you logged in with, and does not equal "-" 



## If this is a bugfix, which Traffic Control versions contained the bug?

Master


## PR submission checklist
